packages/state-manager/src/sagas/communities/updateCommunityMetadata/updateCommunityMetadata.saga.ts

```diff
@@ -34,9 +35,13 @@ export function* sendCommunityMetadataSaga(
     rootCa: community.rootCa,
   }
 
-  yield* apply(
+  const meta = yield* apply(
     socket,
-    socket.emit,
+    socket.emitWithAck,
```

Comment:
This demonstrates the socket.io acknowledgement feature on the frontend. Pretty simple.